File: C#/Aplicatii client-server/labClientServerCSharpProtoBuf/ClientServer/Networking/dto/CursaDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Networking.dto
{
    [Serializable]
    public class CursaDTO
    {
        private string id;
        private int nrPersoane;
        private int capMotor;

        public CursaDTO(string id, int nrPersoane, int capMotor)
        {
            this.id = id;
            this.nrPersoane = nrPersoane;
            this.capMotor = capMotor;
        }

        public virtual string Id { get { return id; } }
        public virtual int NrPersoane { get { return nrPersoane; } }
        public virtual int CapMotor { get { return capMotor; } }
    }
}

File: Java/Aplicatii client-server/labJavaClientServerProtoBuf/Networking/src/main/java/dto/CursaDTO.java
package dto;

import java.io.Serializable;

public class CursaDTO implements Serializable {
    private String id;
    private Integer nrPersoane;
    private Integer capMotor;

    public CursaDTO(String id, Integer nrPersoane, Integer capMotor) {
        this.id = id;
        this.nrPersoane = nrPersoane;
        this.capMotor = capMotor;
    }

    public String getId() {
        return id;
    }

    public Integer getNrPersoane() {
        return nrPersoane;
    }

    public Integer getCapMotor() {
        return capMotor;
    }

    @Override
    public String toString() {
        return "CursaDTO{" +
                "id='" + id + '\'' +
                ", nrPersoane='" + nrPersoane + '\'' +
                ", capMotor=" + capMotor +
                '}';
    }
}
